bin/ci

```diff
@@ -4,6 +4,5 @@ set -exu
 docker-compose build
 docker-compose run --no-deps -e CI=true job './bin/lint'
 docker-compose run -w /work job
-docker-compose run -w /work contracts
```

Comment:
Ahh didn't see this before, I think this is what @posix4e is referring to

Reply:
Hmm, I don't think so @gaieges. Anyway, I am running all tests, this was just a leftover from the previous hmt configuration.